docs/about/history.rst

```diff
@@ -18,6 +18,8 @@ Breaking
 .. bullet_list::
 
    -  Set minimum Unity version to 2021.3.3.
+   -  Moved *Default Clipping State* from *Ocean Renderer* into *Sim Settings Clip Surface* asset.
+      There is no migration available.
```

Comment:
This could probably instruct the user what to do.. maybe something along the lines of this?

"There is no migration available for this setting so if the *Everything Clipped* setting is needed this will need to be configured in the settings data object."

Reply:
Good idea. Done. Thanks.